packages/traceability-tests/tests/issue-verify.test.js

```diff
@@ -0,0 +1,82 @@
+const jose = require('jose');
+const fs = require('fs');
+const { v4: uuidv4 } = require('uuid');
+// pending merge of https://github.com/w3c-ccg/traceability-vocab/pull/735
```

Comment:
putting a comment here to make sure this gets changed after pr 735 is merged but before this pr is.